testgrid/web/src/components/views/InstanceTable.jsx

```diff
@@ -278,6 +296,12 @@ export default class InstanceTable extends React.Component {
         <tr key={kurlUrl}>
           <td>
             <span className="url" onClick={() => this.viewInstanceInstaller(this.props.instancesMap[kurlUrl][0])}>{kurlUrl}</span>
+            {this.props.instancesMap[kurlUrl][0].kurlFlags &&
+              <div>
+                <span>{' -> '}</span>
+                <span className="url" onClick={() => this.viewFlags(this.props.instancesMap[kurlUrl][0])}>{this.props.instancesMap[kurlUrl][0].kurlFlags}</span>
+              </div>
+              }
```

Comment:
Do you think there is an advantage for opening the modal here? 

```suggestion
            {this.props.instancesMap[kurlUrl][0].kurlFlags &&
              <span> [{this.props.instancesMap[kurlUrl][0].kurlFlags}] </span>
              }
```

You need to remove the state change if you opt for this.